azext_iot/digitaltwins/_help.py

```diff
@@ -140,6 +140,112 @@ def load_digitaltwins_help():
             az dt reset -n {instance_name}
     """
 
+    helps["dt data-history"] = """
+        type: group
+        short-summary: Manage and configure data history connections.
+    """
+
+    helps["dt data-history create"] = """
+        type: group
+        short-summary: Creates a data history connection between a Digital Twins instance and another resource.
+    """
+
+    helps["dt data-history create adx"] = """
+        type: command
+        short-summary: Creates a data-history connection between a Digital Twins instance and an Azure Database Explorer.
```

Comment:
It would be helpful if the command description specified the role assignments the CLI will apply in order for the service to successfully create the connection. The long-summary could be used for this.